demisto_sdk/scripts/validate_content_path.py

```diff
@@ -260,11 +264,15 @@ def _validate(path: Path) -> None:
         ):
             raise InvalidLayoutFileName
 
-    if depth == 2 and first_level_folder in {
-        ContentType.INTEGRATION.as_folder,
-        ContentType.SCRIPT.as_folder,
-    }:
-        _validate_integration_script_file(path, parts_after_packs)
+    if depth == 2:
+        if first_level_folder in {
+            ContentType.INTEGRATION.as_folder,
+            ContentType.SCRIPT.as_folder,
+        }:
+            _validate_integration_script_file(path, parts_after_packs)
+        elif first_level_folder in {ContentType.XDRC_TEMPLATE.as_folder}:
+            if path.stem != path.parent.name:
```

Comment:
can we check the suffix here too? while we're there

Reply:
I believe all the files under the **XDRCTemplates** folder should be either json or YAML. Should we check if the suffix is either one?

Reply:
> I believe all the files under the **XDRCTemplates** folder should be either json or YAML. Should we check if the suffix is either one?

yeah, like we do for integrations.